prime-router/docs/proposals/0017-metadata-tables-cleanup.md

```diff
@@ -0,0 +1,441 @@
+    , specimen_source_site              varchar(512) null
+    , specimen_set_id                   int not null default 1
+);
+
+comment on table elr_message_specimen is
+    'ELR messages can theoretically have more than one specimen, so we capture them all here';
+comment on column elr_message_specimen.elr_message_id is
+    'The message ID the specimen belongs to';
+comment on column elr_message_specimen.test_order_id is
+    'The specific test order this specimen is associate with';
+comment on column elr_message_specimen.specimen_id is
+    'The ID for the specimen from the lab';
+comment on column elr_message_specimen.specimen_collection_date_time is
+    'The date time for when the specimen was collected by the ordering facility.';
+comment on column elr_message_specimen.specimen_received_date_time is
+    'The date time for when the specimen was received by the testing facility. For non-traditional sites, this could be the same as the collection date time.';
+comment on column elr_message_specimen.specimen_collection_method_code is
+    'How the specimen was collected. This is the code value we were sent, for example SNOMED, LOINC, etc';
+comment on column elr_message_specimen.specimen_collection_method is
+    'How the specimen was collected. This is the text value we were sent';
+comment on column elr_message_specimen.specimen_collection_site_code is
+    'Where the specimen was collected from. This is the code value we were sent, for example SNOMED, LOINC, etc';
+comment on column elr_message_specimen.specimen_collection_site is
+    'Where the specimen was collected from. This is the text value we were sent.';
+comment on column elr_message_specimen.specimen_type_code is
+    'The type of specimen collected. This is the code value we were sent, for example SNOMED, LOINC, etc';
+comment on column elr_message_specimen.specimen_type is
+    'The type of the specimen collected. This the text value were sent.';
+comment on column elr_message_specimen.specimen_source_site_code is
+    'The source of the specimen collected. This is different from the collection site in that this is where the actual specimen was drawn from, vs the anatomical location. This is the code value we were sent, for example SNOMED, LOINC, etc';
+comment on column elr_message_specimen.specimen_source_site is
+    'The source of the specimen collected. This is different from the collection site in that this is where the actual specimen was drawn from, vs the anatomical location.';
+comment on column elr_message_specimen.specimen_set_id is
+    'The index for the specimen segment. In most cases, this will just be one, but we may get multiple specimens in a message';
+
+-- add our primary key
+alter table elr_message_specimen
+    add constraint elr_message_specimen_pk
+        primary key (elr_message_id, test_order_id, specimen_id)
+;
+
+
+-- the facilities for an elr message
+create table elr_message_facilities
```

Comment:
BLAH.   This seems way too complicated.  One message has multiple facilities?  BLAH.